states/south_dakota/elections.toml

```diff
@@ -3,12 +3,41 @@ date = 2020-06-02
 name = "Congressional & Presidential Primary"
 original_date = 2020-06-02
 
-[20200811]
-date = 2020-08-11
-name = "Congressional Primary Run-Off (if needed)"
-original_date = 2020-08-11
+# [20200811]
+# date = 2020-08-11
+# name = "Congressional Primary Run-Off (if needed)"
+# original_date = 2020-08-11
+# Not Needed this election cycle
 
 [20201103]
 date = 2020-11-03
 name = "General"
 original_date = 2020-11-03
+
+[20201103.registration]
+recieved_by = 2020-10-19T17:00:00
+sources = [
+    "https://sdsos.gov/elections-voting/assets/2020ElectionCALENDAR.pdf",
+    "https://sdsos.gov/elections-voting/voting/register-to-vote/default.aspx",
+    "https://sdsos.gov/elections-voting/upcoming-elections/general-information/2020-Election-Information.aspx"
+]
+
+[20201103.absentee.application]
+in_person_by = 2020-08-10T17:00:00
+recieved_by = 2020-08-10T17:00:00
+sources = [
+    "https://sdsos.gov/elections-voting/assets/2020ElectionCALENDAR.pdf",
+    "https://sdsos.gov/elections-voting/voting/absentee-voting.aspx"
+]
+
+[20201103.absentee]
+in_person_starts = 2020-09-18
```

Comment:
Sure this is in person? seems like this is by mail

Reply:
Hmm... kind of unclear 
"Registered voters can vote in-person once Absentee Voting begins at their County Auditor's office by bringing along a valid photo identification card (ID). If a voter does not have a photo ID, they must be given the option to sign a personal identification affidavit and vote a regular ballot. "

So are they showing up and doing an in person absentee or is it early polling? Hard to say, they kind of mingle the processes into one.

Reply:
Ah, in person makes sense in this case. I was just thinking it was mailed out.